app/app.go

```diff
@@ -81,46 +83,36 @@ import (
 	upgradekeeper "github.com/cosmos/cosmos-sdk/x/upgrade/keeper"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 	appparams "github.com/crypto-com/chain-main/app/params"
-	chainbank "github.com/crypto-com/chain-main/x/bank"
 	"github.com/crypto-com/chain-main/x/chainmain"
 	chainmainkeeper "github.com/crypto-com/chain-main/x/chainmain/keeper"
 	chainmaintypes "github.com/crypto-com/chain-main/x/chainmain/types"
-	chaingov "github.com/crypto-com/chain-main/x/gov"
-	chainstaking "github.com/crypto-com/chain-main/x/staking"
 	supply "github.com/crypto-com/chain-main/x/supply"
 	supplykeeper "github.com/crypto-com/chain-main/x/supply/keeper"
 	supplytypes "github.com/crypto-com/chain-main/x/supply/types"
 
-	appconfig "github.com/crypto-com/chain-main/config"
-
 	// unnamed import of statik for swagger UI support
 	_ "github.com/cosmos/cosmos-sdk/client/docs/statik"
 )
 
+const appName = "chain-maind"
+
 var (
 	// DefaultNodeHome default home directories for the application daemon
-	DefaultNodeHome = func(appName string) string {
-		return os.ExpandEnv("$HOME/.chain-maind")
-	}
-
-	// DefaultCoinParser converts coin denominations
-	DefaultCoinParser = NewSDKCoinParser(appconfig.BaseCoinUnit, appconfig.CoinToBaseUnitMuls)
+	DefaultNodeHome string
 
 	// ModuleBasics defines the module BasicManager is in charge of setting up basic,
 	// non-dependant module elements, such as codec registration
 	// and genesis verification.
 	ModuleBasics = module.NewBasicManager(
 		auth.AppModuleBasic{},
 		genutil.AppModuleBasic{},
-		chainbank.NewAppModuleBasic(DefaultCoinParser),
+		bank.AppModuleBasic{},
```

Comment:
Coin parsing will be done by upstream?

Reply:
yes, it's been merged upstream

Reply:
it's called as `sdk.ParseCoinsNormalized` -- https://github.com/crypto-com/chain-main/pull/275